src/lib.rs

```diff
@@ -71,6 +72,7 @@ pub extern "C" fn exported(
 
     match match command {
         "bh" | "bounty" | "bhunter" | "bountyhunter" => bh::lookup(query, author, rsn_n),
+        "boost" | "boosts" => boost::boost(query),
```

Comment:
This is good but we also need to add `boost` to the triggers and the help output, otherwise this won't trigger & people won't know it's there. Line 106 is where the trigger goes, and about line 130 will be where the helper text goes.

Reply:
Got ya!